Dapper/SqlMapper.cs

```diff
@@ -3608,64 +3611,6 @@ private static MethodInfo ResolveOperator(MethodInfo[] methods, Type from, Type
             return null;
         }
 
-        private static void LoadLocal(ILGenerator il, int index)

```

Comment:
These methods duplicate code that has been part of the BCL:
https://referencesource.microsoft.com/#mscorlib/system/reflection/emit/ilgenerator.cs,d4860f97f6e651d9
https://source.dot.net/#System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.cs,d4860f97f6e651d9